### PR TITLE
[ru] Localize multitenancy.md

### LINKED ISSUE #3244: [ru] Localize `Multitenancy`
### Description

This Issue is to localize [`Multitenancy`](https://github.com/cncf/glossary/blob/main/content/en/multitenancy.md) into Russian.

### Checklist before opening your PR (put `x` in the checkboxes) — note that you have to _add this checklist to your PR and check all the boxes for it to be approved_.
- [] This PR uses the **dev-ru** branch as a source/target.
- [] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.
    
**ETA: TBD**

Useful links:
[Как внести свой вклад](https://glossary.cncf.io/ru/contribute/)
[Руководство по стилю](https://glossary.cncf.io/ru/style-guide/)

[Slack channel](https://app.slack.com/client/T08PSQ7BQ/C05G46RMQTX)
[Telegram channel](https://t.me/ru_cloudnative)

### PR DESCRIPTION
**Pull request by Squoresha**
### Describe your changes
Translate [`Multitenancy`](https://github.com/cncf/glossary/blob/main/content/en/multitenancy.md)

### Related issue number or link (ex: `resolves #issue-number`)
resolves #3244 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/ru/multitenancy.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: Мультитенантность
+status: Completed
+category: Property
+tags: ["architecture", "property", ""]
+---
+
+Мультитенантность (или мультиарендность) — это концепция, при которой одна установка программного обеспечения обслуживает несколько арендаторов.
+Арендатором может быть пользователь, приложение или группа пользователей/приложений, которые работают со своими собственными данными.
+Эти арендаторы не делятся данными между собой (если только владелец не укажет иное) и могут даже не знать о существовании друг друга.
```
Comment (kirkonru) [inline content/ru/multitenancy.md:10]:
```suggestion
Эти арендаторы не обмениваются данными (если только владелец не укажет иное) и могут даже не знать о существовании друг друга.
```

File: content/ru/multitenancy.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+title: Мультитенантность
+status: Completed
+category: Property
+tags: ["architecture", "property", ""]
+---
+
+Мультитенантность (или мультиарендность) — это концепция, при которой одна установка программного обеспечения обслуживает несколько арендаторов.
```
Comment (kirkonru) [inline content/ru/multitenancy.md:8]:
```suggestion
Мультитенантность (или мультиарендность) — это концепция, при которой одна инсталляция программного обеспечения обслуживает несколько арендаторов (тенантов).
```

Comment (kirkonru) [reply content/ru/multitenancy.md:8]: Decided to add _тенантов_ to make things more clear.

File: content/ru/multitenancy.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+Арендатором может быть пользователь, приложение или группа пользователей/приложений, которые работают со своими собственными данными.
+Эти арендаторы не делятся данными между собой (если только владелец не укажет иное) и могут даже не знать о существовании друг друга.
+
+Арендатор может быть как отдельным пользователем с одной учетной записью — например, в случае использования программного обеспечения 
+для решения персональных задач — так и целой корпорацией с тысячами учетных записей, каждая из которых имеет свои права доступа,
+но взаимосвязанных между собой.
```
Comment (kirkonru) [inline content/ru/multitenancy.md:14]:
```suggestion
для решения персональных задач — так и целой корпорацией с тысячами учетных записей, у каждой из которых свои права доступа,
но при этом они взаимосвязаны между собой множеством способов. 
```

File: content/ru/multitenancy.md (hunk 2 above)
Comment (kirkonru) [reply content/ru/multitenancy.md:8]: While _установка_ is perfectly suitable here, to me it is more about the process than the outcome. I'd love to discuss this.

File: content/ru/multitenancy.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+Арендатор может быть как отдельным пользователем с одной учетной записью — например, в случае использования программного обеспечения 
+для решения персональных задач — так и целой корпорацией с тысячами учетных записей, каждая из которых имеет свои права доступа,
+но взаимосвязанных между собой. 
+Такие программные средства как Google Mail, Google Docs, Microsoft Office 365, Salesforce CRM, Dropbox, а также многие другие,
+можно отнести к полностью или частично мультитенантному программному обеспечению.
+
+## Какую проблему решает
+
+Без мультитенантности каждому арендатору требовалась бы отдельная установка программного обеспечения.
+Это увеличивает использование ресурсов и затраты на обслуживание, и как итог — стоимость самого программного обеспечения.
```
Comment (kirkonru) [inline content/ru/multitenancy.md:21]:
```suggestion
Без мультитенантности каждому арендатору требовалась бы отдельная инсталляция программного обеспечения.
Это увеличивало бы потребление ресурсов и затраты на обслуживание, а в конечном итоге и расходы на программное обеспечение.
```

File: content/ru/multitenancy.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+## Как именно решает проблему
+
+Мультитенантное программное обеспечение предоставляет каждому арендатору изолированную среду (рабочие данные, настройки, список учетных данных и т. д.),
+одновременно обслуживая нескольких арендаторов. 
+С точки зрения арендатора, у каждого есть своя собственная установка программного обеспечения, хотя на самом деле они все используют одну.
```
Comment (kirkonru) [inline content/ru/multitenancy.md:27]:
```suggestion
С точки зрения арендатора это выглядит как отдельная инсталляция программного обеспечения, хотя на самом деле все они используют одну и ту же.
```

File: content/ru/multitenancy.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+Это достигается за счет запуска программного средства на сервере и предоставления арендаторам возможности подключаться к нему через сеть, 
+используя интерфейс [API](/application-programming-interface/) (смотри также [Клиент-серверная архитектура](/client-server-architecture/)).
```
Comment (kirkonru) [inline content/ru/multitenancy.md:29]:
```suggestion
Для этого программное средство запускается на сервере, а арендаторам предоставляется возможность подключаться к нему через сеть, 
используя интерфейс [API](/ru/application-programming-interface/) (смотри также [Архитектура клиент-сервер](/ru/client-server-architecture/)).
```

File: content/ru/multitenancy.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+В мультитенантных программным средствах арендаторы делят ресурсы одной установки без влияния друг на друга,
+или взаимодействуя только в соответствии с заранее согласованными условиями.
+Полученная экономия ресурсов на стороне поставщика позволяет значительно снизить стоимость программного обеспечения для пользователей (например, веб-почта или редакторы документов).
```
Comment (kirkonru) [inline content/ru/multitenancy.md:32]:
```suggestion
В мультитенантных программным системах арендаторы делят ресурсы одной инсталляции, не влияя друг на друга,
или взаимодействуя только заранее согласованным, контролируемым образом.
Полученная экономия ресурсов на стороне поставщика позволяет значительно снизить расходы на программное обеспечение для пользователей (хорошие примеры — веб-почта или онлайн-редакторы документов).
```

File: content/ru/multitenancy.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+## Связанные термины
+
+Мультитенантность не является синонимом SaaS, хотя для SaaS очень распространено быть мультитенантным и даже представлять мультитенантность 
+как одно из своих ключевых преимуществ.
```
Comment (kirkonru) [inline content/ru/multitenancy.md:37]:
```suggestion
Мультитенантность не является синонимом SaaS, хотя SaaS-системы часто проектируются как мультитенантные, а сама мультитенантность рекламируется
как одно из их ключевых преимуществ.
```

**Review by kirkonru — COMMENTED** [review]
Thanks for your contribution! 🫶  Just a few itsy-bitsy suggestions from me.

**Comment by Squoresha** [comment]
Thanks for your suggestions! Please check it again

File: content/ru/multitenancy.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+Без мультитенантности каждому арендатору требовалась бы отдельная инсталляция программного обеспечения.
+Это увеличивало бы потребление ресурсов и затраты на обслуживание, а в конечном итоге и расходы на программное обеспечение.
+
+## Как именно решает проблему
+
+Мультитенантное программное обеспечение предоставляет каждому арендатору изолированную среду (рабочие данные, настройки, список учетных данных и т. д.),
+одновременно обслуживая нескольких арендаторов. 
+С точки зрения арендатора это выглядит как отдельная инсталляция программного обеспечения, хотя на самом деле все они используют одну и ту же.
+Для этого программное средство запускается на сервере, а арендаторам предоставляется возможность подключаться к нему через сеть, 
+используя интерфейс [API](/ru/application-programming-interface/) (смотри также [Архитектура клиент-сервер](/ru/client-server-architecture/)).
```
Comment (shurup) [inline content/ru/multitenancy.md:29]: An important detail is missing here. The original article states "via an interface and/or an API".
```suggestion
используя пользовательский интерфейс и/или программный ([API](/ru/application-programming-interface/); смотри также [Архитектура клиент-сервер](/ru/client-server-architecture/)).
```

File: content/ru/multitenancy.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+---
+
+Мультитенантность (или мультиарендность) — это концепция, при которой одна инсталляция программного обеспечения обслуживает несколько арендаторов (тенантов).
+Арендатором может быть пользователь, приложение или группа пользователей/приложений, которые работают со своими собственными данными.
+Эти арендаторы не обмениваются данными (если только владелец не укажет иное) и могут даже не знать о существовании друг друга.
+
+Арендатор может быть как отдельным пользователем с одной учетной записью — например, в случае использования программного обеспечения 
+для решения персональных задач — так и целой корпорацией с тысячами учетных записей, у каждой из которых свои права доступа,
+но при этом они взаимосвязаны между собой множеством способов. 
+Такие программные средства как Google Mail, Google Docs, Microsoft Office 365, Salesforce CRM, Dropbox, а также многие другие,
```
Comment (shurup) [inline content/ru/multitenancy.md:15]:
```suggestion
Такие программные средства, как Google Mail, Google Docs, Microsoft Office 365, Salesforce CRM, Dropbox, а также многие другие,
```

File: content/ru/multitenancy.md [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+Это увеличивало бы потребление ресурсов и затраты на обслуживание, а в конечном итоге и расходы на программное обеспечение.
+
+## Как именно решает проблему
+
+Мультитенантное программное обеспечение предоставляет каждому арендатору изолированную среду (рабочие данные, настройки, список учетных данных и т. д.),
+одновременно обслуживая нескольких арендаторов. 
+С точки зрения арендатора это выглядит как отдельная инсталляция программного обеспечения, хотя на самом деле все они используют одну и ту же.
+Для этого программное средство запускается на сервере, а арендаторам предоставляется возможность подключаться к нему через сеть, 
+используя интерфейс [API](/ru/application-programming-interface/) (смотри также [Архитектура клиент-сервер](/ru/client-server-architecture/)).
+В мультитенантных программным системах арендаторы делят ресурсы одной инсталляции, не влияя друг на друга,
```
Comment (shurup) [inline content/ru/multitenancy.md:30]:
```suggestion
В мультитенантных программным системах арендаторы делят ресурсы одной инсталляции, не влияя друг на друга
```

File: content/ru/multitenancy.md [hunk 12]
```diff
@@ -0,0 +1,37 @@
...
+category: Property
+tags: ["architecture", "property", ""]
+---
+
+Мультитенантность (или мультиарендность) — это концепция, при которой одна инсталляция программного обеспечения обслуживает несколько арендаторов (тенантов).
+Арендатором может быть пользователь, приложение или группа пользователей/приложений, которые работают со своими собственными данными.
+Эти арендаторы не обмениваются данными (если только владелец не укажет иное) и могут даже не знать о существовании друг друга.
+
+Арендатор может быть как отдельным пользователем с одной учетной записью — например, в случае использования программного обеспечения 
+для решения персональных задач — так и целой корпорацией с тысячами учетных записей, у каждой из которых свои права доступа,
```
Comment (shurup) [inline content/ru/multitenancy.md:13]:
```suggestion
для решения персональных задач, — так и целой корпорацией с тысячами учетных записей, у каждой из которых свои права доступа,
```

**Review by shurup — COMMENTED** [review]
Thank you for this translation! There are just a few minor suggestions from me.

**Comment by Squoresha** [comment]
Thanks for your suggestions! Please check it again

**Review by shurup — APPROVED** [review]
Thank you, LGTM now! We only need to squash it before merging.

**Review by kirkonru — APPROVED** [review]

**Pull request merged**